src/render/VolumeRaycastShader.cpp

```diff
@@ -54,186 +54,183 @@ VolumeRaycastShader::VolumeRaycastShader() : m_use_blending{false}, m_blend_volu
                    "}\n";
 
   m_fragmentShader =
```

Comment:
Is it possible to load the shaders from a text file like in the web volume viewer? Would be a decent add if it's not too much work

Reply:
https://github.com/brown-ccv/VR-Volumeviewer/issues/53